packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archives.constants.js

```diff
@@ -29,17 +29,31 @@ export const CHECK_PRICES_DOC_LINK = {
 
 export const GUIDES = [
   {
-    id: 'cold-archive-getting-started',
-    link:
-      'https://docs.ovh.com/fr/storage/s3/getting-started-with-cold-archive/',
+    id: 'overview',
+    links: {
+      default:
+        'https://docs.ovh.com/en/storage/object-storage/cold-archive/overview/',
+      fr:
+        'https://docs.ovh.com/fr/storage/object-storage/cold-archive/overview/',
+    },
```

Comment:
i guess it's a good practice to write them in capital to indicate they are fixed values (constant).
```suggestion
    LINKS: {
      DEFAULT:
        'https://docs.ovh.com/en/storage/object-storage/cold-archive/overview/',
      FR:
        'https://docs.ovh.com/fr/storage/object-storage/cold-archive/overview/',
    },
```

Reply:
fixed.